README.md

```diff
@@ -1,101 +1,93 @@
-# Progetto Spring Boot - Gestione Autosalone
+# Progetto: Piattaforma di Streaming con Acquisto e Noleggio di Film
+Questo progetto mira a sviluppare una piattaforma di streaming, simile all'app TV Show Time, che consente agli utenti di gestire la propria esperienza di visione di film e serie TV, con l'aggiunta di funzionalità per acquistare e noleggiare contenuti.
 
-## Descrizione del progetto
-
-L’autosalone avrà degli utenti semplici (acquirenti) che possono effettuare l’ordine o l’acquisto di un modello in pronta consegna. O il noleggio di un veicolo. Avrà degli utenti di amministrazione (admin) che potranno aggiungere veicoli in pronta consegna, usati, ordinabili, noleggiabili e potranno gestire le vendite i noleggi e gli ordini. Avrà poi dei venditori che gestiranno le vendite e i noleggi.
+Aggiungere ruolo così da avere una sola classe utente che poi si differenzia per metodi in base al ruolo
 
 ## Utenti
-
-### Acquirente
-
-Per ogni acquirente avremo una serie di attributi:
-
+### Utente
+Per ogni Utente avremo una serie di attributi:
+- Id
 - Nome
 - Cognome
 - Telefono
 - Email
 - Password
-
-### Amministratore
-
-Per ogni amministratore avremo una serie di attributi:
-
-- Nome
-- Cognome
-- Email
-- Password
-
-### Venditore
-
-Per ogni venditore avremo una serie di attributi:
-
-- Nome
-- Cognome
-- Telefono
-- Email
-- Password
-
-## Veicoli
-
-I veicoli possono essere di diversi tipi: auto, moto, scooter, furgoni. Per ogni veicolo avremo una serie di attributi:
-
-- Marca
-- Modello
-- Cilindrata
-- Colore
-- Potenza
-- Tipo Cambio
-- Anno immatricolazione
-- Alimentazione
-- Prezzo
-- Eventuale sconto sul prezzo di listino
-- Eventuali accessori a corredo
-- Flag che identifichi se il veicolo è nuovo o usato
-- Flag che identifichi se il veicolo è ordinabile, acquistabile o non più disponibile
-
-## Ordini/Acquisti
-
-Per ogni ordine/acquisto avremo:
-
-- Anticipo
+- Ruolo(User o Admin)
+- Status Abbonamento
+- Lista di film/serie già viste
+- Lista di film/serie da vedere
+
+
+
+## Film/Serie TV
+I film possono essere di diversi tipi: horror, romantico, commedia, ecc (così come le serie)
+Per ogni Film avremo una serie di attributi:
+- Id
+- Titolo
+- Durata
+- Genere
+- Regista
+- Cast
+- Anno di uscita
+- Se è conclusa oppure no (serie tv)
+- Rating iMdb
+- Prezzo acquisto
+- Prezzo noleggio
+- Trama/Descrizione
+- Suggeriti (?)
```

Comment:
Come mai questi punti interrogativi?

Reply:
Era un promemoria personale per le implementazioni "extra" e da valutare in corso d'opera con Lorenzo, posso toglierli :)